src/plugins/api-proxy/proxy.js

```diff
@@ -1,53 +1,34 @@
+// @flow
 // Implicit dependency: tracking plugin providing request.trackApi method
 import { Observable } from 'rxjs/Observable';
 import 'rxjs/add/observable/zip';
 
 import {
 	buildRequestArgs,
 	makeApiRequest$,
-	parseRequest,
-} from '../util/apiUtils';
+	getExternalRequestOpts,
```

Comment:
`parseRequest` was slightly refactored to _only_ generate `externalRequestOpts`, which are the 'options' supplied to the `request` call to the REST API - essentially porting the incoming request attributes to the outgoing request.